resources/packs/os/services/sysv.go

```diff
@@ -34,7 +34,7 @@ func (s *SysVServiceManager) List() ([]*Service, error) {
 	}
 
 	// eg. we ignore the following run levels since `service halt status` may shutdown the system
-	ignored := []string{"functions", "killall", "halt", "reboot", "shutdown"}
+	ignored := []string{"functions", "killall", "halt", "boot", "shutdown"}
```

Comment:
I think we need both, "reboot" and "boot" since we just ignore the specific services, if I remember correctly "reboot" is used on older ubuntu machines.

Reply:
That is true.
I added the files mentioned in the SUSE boot readme file. Most of these would not handle a call like `service x status` properly and are used for booting, switching run levels, or killing processes. (The list is also sorted now because it became quite long)
```suggestion
	ignored := []string{"boot", "boot.local", "functions", "halt", "halt.local", "killall", "rc", "reboot", "shutdown", "single", "skeleton", ".depend.boot", ".depend.start", ".depend.stop" }
```